### PR TITLE
puts modified and added formulae

### PR DESCRIPTION
**Pull request by ladislas**
kind of like #258, this PR outputs the added and modified formulae to help debug.

I could also add `deleted_formulae` if needed.

Let me know what you think.

### CODE REVIEW

**Comment by sjackman** [comment]
I scanned the code quickly, and I didn't see that the value of `@formulae @added_formulae @modified_formulae` are printed anywhere. If that's the case, :+1: from me.

**Comment by iMichka** [comment]
I like too. Nice addition.

Can we add the deleted formulae too?

File: cmd/brew-test-bot.rb [hunk 1]
```diff
@@ -574,6 +574,13 @@ def download
       @formulae += @added_formulae + @modified_formulae
       @test_brew = (!@tap || @test_bot_tap) &&
                    (@formulae.empty? || @test_default_formula)
+
+      puts <<~EOS
+
+        Bottles to be built for:
+          added formulae    #{@added_formulae.empty? ? "(empty)" : @added_formulae}
```
Comment (MikeMcQuaid) [inline cmd/brew-test-bot.rb]: Use `blank?` rather than `empty?`

File: cmd/brew-test-bot.rb [hunk 2]
```diff
@@ -574,6 +574,13 @@ def download
       @formulae += @added_formulae + @modified_formulae
       @test_brew = (!@tap || @test_bot_tap) &&
                    (@formulae.empty? || @test_default_formula)
+
+      puts <<~EOS
+
+        Bottles to be built for:
```
Comment (MikeMcQuaid) [inline cmd/brew-test-bot.rb]: Bottles aren't necessarily going to be built. How about `Formula changes to be tested:`

File: cmd/brew-test-bot.rb (hunk 1 above)
Comment (ladislas) [reply cmd/brew-test-bot.rb]: sure!

File: cmd/brew-test-bot.rb (hunk 2 above)
Comment (ladislas) [reply cmd/brew-test-bot.rb]: Sounds good to me, I'll change that.

**Comment by ladislas** [comment]
I just updated the PR based on your feedback.

I've added `deleted_formulae`.

it's gonna print as `["formula1", "formula2"]` -- should it be printed as `formula1, formula2` instead?

**Comment by sjackman** [comment]
> it's gonna print as ["formula1", "formula2"] -- should it be printed as formula1, formula2 instead?

Yes, separated with either comma space `, ` or simply space ` `

**Comment by ladislas** [comment]
@sjackman should I use:

```ruby
@modified_formulae.join(' ') # => "formula1 formula2"

#or 

@modified_formulae.reject(&:blank?).join(' ') # => same but will remove '' and nil, just in case
```

**Comment by ladislas** [comment]
I went with the safer `reject`, to handle all cases.

**Comment by sjackman** [comment]
There will not be blank formulae names, so `@modified_formulae.join(' ')` is fine.

**Comment by MikeMcQuaid** [comment]
> There will not be blank formulae names, so `@modified_formulae.join(' ')` is fine.

Agreed. Once this is done this should be good to go!

**Comment by ladislas** [comment]
Done! ✅ (from my phone for the first time)

**Comment by sjackman** [comment]
> Done! ✅ (from my phone for the first time)

Hah. Well done! Thanks again for this work, Ladislas! 🌟

**Comment by MikeMcQuaid** [comment]
Thanks again @ladislas!

**Comment by MikeMcQuaid** [comment]
@ladislas Please check your email 😂

**Pull request merged**